src/config/rollup.ts

```diff
@@ -32,10 +32,11 @@ export default {
 			};
 		},
 
-		output: (): OutputOptions => {
+		output: (opts): OutputOptions => {
```

Comment:
What's the intention here? That people specify `output: config.server.output({ format: 'esm' })` in their `rollup.config.js`? This should be documented somewhere, and included in the types, if do.

Is it possible to have this also use the `pkg.type` value instead of making users have to configure it?